packages/crypto/src/browser/cryptoApiWrapper.ts

```diff
@@ -0,0 +1,15 @@
+import type { CryptoApi } from '../types';
+
+import { bufferFromString, stringFromBuffer, decrypt, encrypt, keyFromPassword } from './aes-ctr';
+import { randomBytes } from './randomBytes';
+
+const cryptoApiWrapper: CryptoApi = {
+  bufferFromString,
+  stringFromBuffer,
+  decrypt,
+  encrypt,
+  keyFromPassword,
+  randomBytes,
+};
+
+export default cryptoApiWrapper;
```

Comment:
Would this be valid? If yes, we could rename this file to `index.ts`.

```suggestion
export default { ...cryptoApiWrapper };
```